M8ag89-d1e60/cod-u8i00i_M8ag89-d1e60.xml

```diff
@@ -57,277 +57,624 @@
-            septemtrionalem lapidis approximes ferrum fugare videbitur quousque pars meridionalis ferro
-            eidem coniungatur Et similiter de parte reliqua idem intelligas Si tamen violentia fiat partibus
-            quod videlicet pars ferri meriodionalis quae cum septemtrionali lapidis tacta fuerit tangatur cum
-            meridionali lapidis vel illa quae cum meridionali tacta fuit quae etiam meridionalis in ferro
-            appellatur meridionali lapidis iungatur alterabitur virtus in ferro de facili et fiet meridionale quod
-            fuit septemtrionale in eo et e converso Et causa huius est impressio ultimi agentis confundentis et
-            alterantis virtutem primi</p>
+          <p xml:id="M8ag89-d1e60-d1e229">
+            <lb ed="#O" n="20"/>Perche la calamita tiri il ferro. Cap.o VII.
+          </p>
+          <p xml:id="M8ag89-d1e60-d1e231">
+            <lb ed="#O" n="21"/>Se tu vuoi secondo il naturale appetito della calamita
+            <!--00000012.xml-->
+            <pb ed="#O" n="185-v"/>
+            <lb ed="#O" n="1"/>tirare il ferro, che è posto sopra il legno à notare nell'
+            <lb ed="#O" n="2"/>aqua mira la parte di tramontana del ferro, et accos¬
+            <lb ed="#O" n="3"/>tagli la parte di ostro della calamita tu ti accorgerai,
+            <lb ed="#O" n="4"/>che ella la seguita, overo simigliantamente, alla
+            <lb ed="#O" n="5"/>parte di ostro del ferro porgi la parte di tramontana
+            <lb ed="#O" n="6"/>della calamita, vederai, che ella senza regola et
+            <lb ed="#O" n="7"/>resistenza la seguita. ché se per lo contrario porgerai
+            <lb ed="#O" n="8"/>alla parte di tramontana della calamita, parera,
+            <lb ed="#O" n="9"/>che ella saci il ferro, et da se lo allontani, fin tanto
+            <lb ed="#O" n="10"/>che la parte di ostro del ferro si congiungera con essolei,
+            <lb ed="#O" n="11"/>et l'istesso intendi della parte restante. Mà se
+            <lb ed="#O" n="12"/>si fara violentia nelle parti si fattamente, che la
+            <lb ed="#O" n="13"/>parte dell'ostro del ferro, la quale si osara toccata
+            <lb ed="#O" n="14"/>con la parte di tramontana della pietra, si tocchi
+            <lb ed="#O" n="15"/>con la parte dell'ostro della pietra: overo questa,
+            <lb ed="#O" n="16"/>che si sara toccata con la parte dell'ostro, si congiunga
+            <lb ed="#O" n="17"/>con la parte dell'ostro della pietra; la virtù
+            <lb ed="#O" n="18"/>nel ferro osi alterera facilmente, et si fara
+            <lb ed="#O" n="19"/>di tramontana quel, che prima era in lui di
+            <lb ed="#O" n="20"/>ostro, et così per lo contrario; la causa di ciò la
+            <lb ed="#O" n="21"/>ultima impressione dell'agente, che confonde, et con¬
+            <lb ed="#O" n="22"/>la virtù del primo motore.
+            <!--00000014.xml correct position in XML?-->
+            <pb ed="#O" n="186-r"/>
```

Comment:
pb should always go in the starting new pargraph rather than at the end of a paragraph